src/main/java/db/migration/V2__init_menu_session.java

```diff
@@ -0,0 +1,26 @@
+package db.migration;
+
+import org.flywaydb.core.api.migration.spring.SpringJdbcMigration;
+import org.springframework.jdbc.core.JdbcTemplate;
+
+/**
+ * Initialize the formplayer session database
+ * @author wspride
+ */
+public class V2__init_menu_session implements SpringJdbcMigration {
+
+    @Override
+    public void migrate(JdbcTemplate jdbcTemplate) throws Exception {
+        jdbcTemplate.execute("CREATE TABLE menu_sessions (\n" +
+                "    id text NOT NULL,\n" +
+                "    username text,\n" +
+                "    domain text,\n" +
```

Comment:
it's good practice to label these as NOT NULL if they are required. that way we never accidentally commit bad data